src/components/organisms/MessageBox/MessageBox.tsx

```diff
@@ -36,7 +38,7 @@ const MessageBox: React.FC = () => {
       // @ts-ignore
       notification[notificationType]({
         key: alert.id,
-        message: alert.title,
+        message: enhanceErrorMessage(alert.title),
```

Comment:
This here shouldn't be enhanced since it's the title, just the description should be modified